File: C#/BluffinMuffin.Protocol/Game/SeatUpdatedCommand.cs
using BluffinMuffin.Protocol.DataTypes;

namespace BluffinMuffin.Protocol.Game
{
    /// <summary>
    /// Server informs clients that a seat has changed state. Usually sent when player sits-in or sits-out.
    /// </summary>
    public class SeatUpdatedCommand : AbstractGameCommand
    {
        /// <summary>
        /// The seat that has changed state
        /// </summary>
        public SeatInfo Seat { get; set; }
    }
}

File: Python/BluffinMuffin.Protocol/game/seat_updated_command.py
from abstract_game_command import AbstractGameCommand
from data_types.seat_info import SeatInfo

__author__ = 'ericmas001@gmail.com'



class SeatUpdatedCommand(AbstractGameCommand):
    def __init__(self, obj):
        super().__init__(obj)
        self.seat = SeatInfo(obj['Seat'])

    def __str__( self ):
        return '{0} ({1})'.format(
            super().__str__(),
            self.seat
        )
